src/ngx_http_lua_kong_module.c

```diff
@@ -128,7 +136,22 @@ ngx_http_lua_kong_create_loc_conf(ngx_conf_t* cf)
         return NULL;
     }
 
+    conf->error_log_append = NGX_CONF_UNSET_PTR;
+
     return conf;
 }
 
 
+static char*
+ngx_http_lua_kong_merge_loc_conf(ngx_conf_t *cf, void *parent, void *child)
+{
+    ngx_http_lua_kong_loc_conf_t *prev = parent;
+    ngx_http_lua_kong_loc_conf_t *conf = child;
+
+    ngx_conf_merge_str_value(conf->tag, prev->tag, "");
```

Comment:
`tag` is location only, should not merge it.

Reply:
yes! thank you for pointing that out.